src/NServiceBus.Core/Features/FeatureConfigurationContext.cs

```diff
@@ -41,6 +41,18 @@ internal FeatureConfigurationContext(ReadOnlySettings settings, IConfigureCompon
 
         internal RoutingComponent Routing { get; }
 
+        internal ReceiveConfiguration Receiving
+        {
+            get
+            {
+                if (receiving == null)
```

Comment:
This feels a bit messy, but I don't immediately have a better suggestion on how to handle it since, send-only endpoints will have the property also...

It could be written a bit more succinctly, though:

```
internal ReceiveConfiguration Receiving => receiving ?? throw new InvalidOperationException("...");
```

Reply:
Done